src/main/webapp/app/faq/faq.service.ts

```
@@ -137,4 +137,16 @@ export class FaqService {
             return categories.some((category) => filteredCategory.has(category!));
         }
     }
+
+    hasSearchTokens(faq: Faq, searchTerm: string) {
+        if (searchTerm == '') {
+            return true;
+        }
+        const tokens = searchTerm.split(' ');
+        if (tokens) {
+            let faqText = faq.questionTitle + ' ' + faq.questionAnswer;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)